tests/ctl/nmstatectl_test.py

```diff
@@ -24,33 +24,54 @@
 from .compat import mock
 
 from nmstatectl import nmstatectl
+from libnmstate.schema import Constants
+from libnmstate.schema import Route
 
-LO_JSON_STATE = """{
+EMPTY_ROUTE_JSON_STATE = """"{ROUTES}": {{
+        "{ROUTE_CONFIG}": [],
+        "{ROUTE_RUNNING}": []
+    }}""".format(ROUTES=Constants.ROUTES,
```

Comment:
If it is a yaml string, I prefer to have it straight clean so it can be copy/pasted.
The point if a yaml in the test is readability.

If we need to build it from constants, then I think it will be better to use a straight dict and not yaml.